dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/CheckpointPhaseTest.java

```diff
@@ -51,12 +49,9 @@ public class CheckpointPhaseTest {
     public TestName testName = new TestName();
     public static final String APP_NAME = "app2";
 
-    @Server("checkpointFATServer")
+    @Server("checkpointPhaseServer")
     public static LibertyServer server;
 
-    @ClassRule
-    public static RepeatTests repeatTest = FATSuite.defaultMPRepeat("checkpointFATServer");
-
```

Comment:
Do we not repeat this test?

Reply:
I did remove the repeat for this test because it is not testing anything interesting from the programming model (EE or MP).  Instead it is testing the before and after app start options and other error conditions like what happens if an application fails to start.  I do not see the need to repeat such a test across different EE versions.

Reply:
Got it, thanks!